pandas/core/indexes/base.py

```diff
@@ -1471,6 +1471,41 @@ def is_object(self):
         return is_object_dtype(self.dtype)
 
     def is_categorical(self):
+        """
+        Check if the Index type is categorical.
+
+        The given object must be a pandas DataFrame or pandas Series.
+
+        Returns
+        -------
+        boolean
+            True if index is categorical.
+
+        Examples
+        --------
+        >>> s = pd.Series(["Watermelon", "Orange", "Apple",
+        ...                 "Watermelon"]).astype('category')
+        >>> df = pd.DataFrame({'Weight (grams)': [2000, 230, 160, 1300]},
```

Comment:
don't create a DataFrame here just to use its index, just use the series above

Reply:
Ok, Thank you! I have changed the examples.